src/component/JoinList.js

```diff
@@ -63,6 +63,7 @@ const JoinList = ({ firestore }) => {
           Join List
         </button>
       </form>
+      <br></br>
```

Comment:
Using `<br>` tags to add space is not a great habit to get into. Non-blocking, but better to remove this and use CSS to add the space you desire